decidim-core/app/views/decidim/account/show.html.erb

```diff
@@ -2,19 +2,19 @@
 <% content_for(:subtitle) { t("profile", scope: "layouts.decidim.user_menu") } %>
 
 <div class="row">
-  <%= decidim_form_for(@account, url: account_path, method: :put, html: { autocomplete: "off" }) do |f| %>
-    <input autocomplete="off" name="hidden" type="password" style="display:none;">
+  <%= decidim_form_for(@account, url: account_path, method: :put, html: { autocomplete: "on" }) do |f| %>
+    <input autocomplete="current-password" class="hide" name="hidden" type="password">
```

Comment:
This originates from #4741. The original issue is explained better at #2529.

If the current version of FF works better than in the times of those PRs, I believe we could remove this input field completely from here. Can you test how this behaves in FF and whether the issue explained at those PRs still persists?

If FF behaves better these days, we can get rid of this line. At least FF 67 seems to have added support for the `new-password` value that you have correctly applied to the password fields and it was released 3 years ago already.

https://bugzilla.mozilla.org/show_bug.cgi?id=1119063
![Firefox milestone](https://user-images.githubusercontent.com/864340/166500851-196f5592-fc25-49c3-b432-1d91d1d868ca.png)